tools/distpartitioning/utils.py

```diff
@@ -689,7 +690,10 @@ def map_partid_rank(partid, world_size):
 
 
 def generate_read_list(num_files, world_size):
-    """Generate the file IDs to read for each rank.
+    """
```

Comment:
how about re-naming this function to indicate what kind of split strategy is used?

Reply:
Resolving to keep conversation in one thread.